steps/fabric-crashlytics-deployer/1.0.0/step.yml

```diff
@@ -0,0 +1,84 @@
+title: Fabric / Crashlytics deployer
+summary: |
+  Deploy to Fabric/Crashlytics beta testing service your apk.
+description: |
+  Deploy to Fabric/Crashlytics beta testing service your apk.
+website: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer
+source_code_url: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer
+support_url: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer/issues
+published_at: 2018-01-20T13:00:17.884460961+01:00
+source:
+  git: https://github.com/manuelgon47/bitrise-step-fabric-crashlytics-deployer.git
+  commit: 8411b25f09ad8018519499410d04e1b2e700cd6f
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fabric_api_key: null
+  opts:
+    description: |
+      Set the fabric api key
+    is_required: true
+    summary: Set the fabric api key
+    title: 'Fabric: API Key'
+- fabric_build_secret: null
+  opts:
+    description: |
+      Set the fabric build secret
+    is_required: true
+    summary: Set the fabric build secret
+    title: 'Fabric: Build Secret'
+- fabric_apk_path: null
```

Comment:
Is it possible to use `$BITRISE_APK_PATH` as the default value?

Reply:
Sure!

Reply:
I open an other PR versioned to 1.0.1 with your change request
https://github.com/bitrise-io/bitrise-steplib/pull/1361